src/auth/credential.ts

```diff
@@ -237,12 +237,9 @@ function requestAccessToken(transit, options: Object, data?: Object): Promise<Go
 export class CertCredential implements Credential {
   private certificate_: Certificate;
 
-  constructor(serviceAccountPathOrObject: string|Object) {
-    if (typeof serviceAccountPathOrObject === 'string') {
-      this.certificate_ = Certificate.fromPath(serviceAccountPathOrObject);
-    } else {
-      this.certificate_ = new Certificate(serviceAccountPathOrObject);
-    }
+  constructor(serviceAccountPathOrObject: string | object) {
+    this.certificate_ = (typeof serviceAccountPathOrObject === 'string')
+      ? Certificate.fromPath(serviceAccountPathOrObject) : new Certificate(serviceAccountPathOrObject);
```

Comment:
Better for readability to have the ? on the previous line

Reply:
Done